libs/ng-strapi-auth/src/lib/ng-strapi-auth-options.ts

```diff
@@ -0,0 +1,13 @@
+import { ToastPosition } from "@ngneat/hot-toast";
+export class AuthOptionModel {
+    appName?: string;
+    baseAPIPath?: string;
+    roleList?: string[];
+    accessRole?: string;
```

Comment:
This can be passed to the authguards directly right ? so we can have multiple roles in the same app